### ISSUE TITLE
[newrelic-logging] Helm Chart has Incomplete Env Variables

### ISSUE THREAD

**Issue by jsubirat** [newrelic/helm-charts#81]
**Bug description**

[Originally posted by @druidsbane in the [newrelic-fluent-bit-output repository](https://github.com/newrelic/newrelic-fluent-bit-output/issues/33)]

**Version of Helm and Kubernetes**

<Please provide this information, @druidsbane, @jasonchester>

**Which chart?**

newrelic-logging (current version)

**What happened?**

The pods were unable to get host status until I added:

```
- name: HOSTNAME
  valueFrom:
    fieldRef:
        apiVersion: v1
        fieldPath: metadata.name
```

This also happened on Azure Kubernetes Service (AKS) and was fixed by the above addition.

Can we get this added to the Helm chart and the regular deployment file?

More details on a fluent-bit issue: fluent/fluent-bit#850 (comment)

**What you expected to happen?**

<Please provide this information, @druidsbane, @jasonchester>

**How to reproduce it?**

<Please provide this information, @druidsbane, @jasonchester>

Kubernetes information as provided by @jasonchester:

```
{Major:"1", Minor:"16", 
GitVersion:"v1.16.7", GitCommit:"e2d9f8479783020904aba3de7499a49be6c75ebd", GitTreeState:"clean",
BuildDate:"2020-04-09T02:31:15Z", 
GoVersion:"go1.13.6", Compiler:"gc", 
Platform:"linux/amd64"}
```

**Anything else we need to know?**

<Please provide this information, @druidsbane, @jasonchester>

**Comment by gungorugur** [comment]
We have same issue. We've added HOSTNAME env but  issue still goes on.  We use aks 1.17.13. Any ideas ?

**Comment by jsubirat** [comment]
Addressed in PR #911
